### DISCUSSION TITLE
怎么使用？

### DISCUSSION THREAD

**Discussion by SpiriteadAway** [thatskymod/Sky-CotL-Scripts#130] [Q&A]
PC怎么下载使用？

**Comment by thatskymod** [comment]
Up-to-date documentation and usage instructions for the PC mod are available at https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/PC#readme (see the "Automated Installation Instructions" down below).

Curious about anything specific? Just let me know!

**Comment by thatskymod** [comment]
Closing due to a lack of activity and specifics in the thread. Feel free to submit new details so we can look into it further!
